index.html

```diff
@@ -0,0 +1,256 @@
+        <div class="workflow_arrow">
+          <i class="fas fa-long-arrow-alt-right fa-lg"></i>
+        </div>
+        <div class="workflow_item">
+          <div class="workflow_image"></div>
+          <div>
+            output <code>.html</code>, <code>.pdf</code>, <code>.docx</code>
+          </div>
+        </div>
+        <div class="workflow_arrow">
+          <i class="fas fa-long-arrow-alt-right fa-lg"></i>
+        </div>
+        <div class="workflow_item">
+          <div class="workflow_image"></div>
+          <div>test, deploy to host</div>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="section_header">The Best Parts</div>
+      <div class="row flex_wrap_reverse">
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-atlas fa-lg"></i> Open, collaborative, versioned
+          </div>
+          <div>
+            Author your manuscript the open-source way.
+            Let anyone propose changes.
+            Iteratively review, discuss, and refine with collaborators.
+            Coordinate teams of any size with ease.
+            Keep track of versions, to-dos, revisions, and who wrote what and when.
+            All in a <a href="https://github.com/">central location</a>.
+          </div>
+        </div>
+        <div class="col_image"></div>
+      </div>
+      <div class="row flex_wrap">
+        <div class="col_image"></div>
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-magic fa-lg"></i> Automated and continuous
+          </div>
+          <div>
+            Never use a reference manager again.
+            Provide a DOI, ISBN, PubMed ID, url, etc., and let Manubot generate and number citations for you.
+            See your manuscript update right away as you make changes. 
+            Test and deploy automatically with <a href="https://travis-ci.org/">continuous integration</a>.
+          </div>
+        </div>
+      </div>
+      <div class="row flex_wrap_reverse">
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-hand-point-up fa-lg"></i> Beautiful and interactive
+          </div>
+          <div>
+            Static is out, dynamic is in.
+            Publish your paper with the contemporary look and interactive features people have come to love and expect from modern webpages: tooltips, public annotations, table of contents, figure viewer, and more.
+          </div>
+        </div>
+        <div class="col_image"></div>
+      </div>
+    </section>
+    <section>
+      <div class="row flex_wrap">
+        <div class="col_image"></div>
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-lock-open fa-lg"></i> Free and open-source
+          </div>
+          <div>
+            There's no need to pay to produce brilliant manuscripts.
+            Manubot is 100% free and open-source.
```

Comment:
This is debatable. From https://greenelab.github.io/meta-review/v/01058fc474786870c07d41e1df961ed30f4436e4/:

> Manubot and its dependencies are free of charge and largely open source. It does rely on gratis services from two proprietary platforms: GitHub and Travis CI. Fortunately, lock-in to these services is minimal, and several substitutes already exist.

I guess it depends on what falls under the Manubot umbrella.